packages/nitro-protocol/test/contracts/MultiAssetHolder/claim.test.ts

```diff
@@ -220,7 +220,7 @@ describe('claim', () => {
             args: {
               channelId: targetId,
               assetIndex: BigNumber.from(0),
-              initialHoldings: heldBefore[addresses.g],
+              initialHoldings: heldBefore[guarantorId],
```

Comment:
From my reading of the file, I can't figure out why (if?) this was necessary.

Reply:
I think it is unnecessary in fact. Probably a debugging artifact.